lib/parser.js

```diff
@@ -576,6 +576,16 @@ const deprecatedArgs = [
           'adb shell am instrument -e coverage true -w ',
   }],
 
+  [['--android-coverage-end-intent'], {
```

Comment:
since this is not an existing CLI param, let's just not include it here or in the command line flag documentation. it should be a capability only.